warehouse/ingest-core/src/test/java/datawave/ingest/table/aggregator/GlobalIndexUidAggregatorExpandedTest.java

```diff
@@ -0,0 +1,719 @@
+                        .withRemovals("uid1", "uid2", "uid3", "uid4", "uid5", "uid6", "uid7", "uid8").build());
+        
+        testCombinations(expectation, asList(value1, value2));
+    }
+    
+    @Test
+    public void unexpectedDataHandling() {
+        // Does a legacy Value (no UIDs, not count-only, yet a count=1) get effectively ignored?
+        Value value1 = UidTestBuilder.newBuilder().withUids().withCountOverride(1).build();
+        
+        Value value2 = UidTestBuilder.newBuilder().withUids("uid1", "uid2").build();
+        
+        Value value3 = UidTestBuilder.newBuilder().withUids().withRemovals("uid1", "uid3", "uid4").build();
+        
+        Uid.List expectation = UidTestBuilder.valueToUidList(UidTestBuilder.newBuilder().withUids("uid2").withRemovals("uid1", "uid3", "uid4").build());
+        
+        testCombinations(expectation, asList(value1, value2, value3));
+    }
+    
+    @Test
+    public void duplicatesRecountedUnderSeenIgnore() {
+        // Does a count-only Value, when seen first, prevent de-duplication of named
+        // UID lists (done for performance optimization)?
+        agg = new GlobalIndexUidAggregator(10);
+        Value value1 = UidTestBuilder.newBuilder().withCountOnly(11).build();
+        
+        Value value2 = UidTestBuilder.newBuilder().withUids("uid1", "uid2").build();
+        
+        Value value3 = UidTestBuilder.newBuilder().withUids("uid1", "uid2").build();
+        
+        Uid.List expectation = UidTestBuilder.valueToUidList(UidTestBuilder.newBuilder().withCountOnly(15).build());
+        
+        // not testing the reversal because its behavior is captured
+        // in testDuplicatesRecountedUnderSeenIgnoreReverse
+        isForwardOnlyTest = true;
+        testCombinations(expectation, asList(value1, value2, value3));
+    }
+    
+    @Test
+    public void duplicatesRecountedUnderSeenIgnoreReverse() {
+        // Does a count-only Value, when seen last, NOT prevent de-duplication of named
+        // UID lists (done for performance optimization)?
+        agg = new GlobalIndexUidAggregator(10);
+        Value value1 = UidTestBuilder.newBuilder().withUids("uid1", "uid2").build();
+        
+        Value value2 = UidTestBuilder.newBuilder().withUids("uid1", "uid2").build();
+        
+        Value value3 = UidTestBuilder.newBuilder().withCountOnly(11).build();
+        
+        Uid.List expectation = UidTestBuilder.valueToUidList(UidTestBuilder.newBuilder().withCountOnly(13).build());
+        
+        // not testing the reversal because its behavior is captured
+        // in testDuplicatesRecountedUnderSeenIgnore
+        isForwardOnlyTest = true;
+        testCombinations(expectation, asList(value1, value2, value3));
+    }
+    
+    @Test
+    public void floorOfZeroWhenSeenIgnoreFalse() {
+        // Does count match size of UID list even when there are more removal UIDs than UIDs?
+        agg = new GlobalIndexUidAggregator(10);
```

Comment:
There's no value in overriding the aggregator here, so this line could be removed, right? Or am I missing something in this test?

Reply:
It seems there are a bunch more of these below, as well.

Reply:
Good catch.  I removed all of the extraneous aggregator overrides.